eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -230,11 +230,61 @@ function Set-ApiViewCommentForPR {
   }
 }
 
-function Save-PackageProperties ($repoRoot, $serviceNames, $outputDirectory) {
-  $scriptPath = Join-Path $repoRoot "eng" "common" "scripts" "Save-Package-Properties.ps1"
-  $serviceDirectories = $serviceNames -split ","
+# Helper function used to create API review requests for Spec generation SDKs pipelines
+function Create-API-Review {
+  param (
+    [string]$apiviewEndpoint = "https://apiview.dev/PullRequest/DetectAPIChanges",
+    [string]$specGenSDKArtifactPath,
+    [string]$apiviewArtifactName,
+    [string]$buildId,
+    [string]$commitish,
+    [string]$repoName,
+    [string]$pullRequestNumber
+  )
+  . ${PSScriptRoot}\..\logging.ps1
+  $specGenSDKContent = Get-Content -Path $SpecGenSDKArtifactPath -Raw | ConvertFrom-Json
```

Comment:
What is in the artifact data? Is it just the standard data from spec-gen-sdk or did we create a custom file just for apiview? I just want to understand how coupled we are going to be to the data structure and property names here.

Reply:
In some cases the artifact is the same a s the SDK Package. in some other cases we are generating the APIView code file (Python, Js)